src/backend/PipelineLayout.cpp

```diff
@@ -34,31 +34,22 @@ namespace backend {
 
     // PipelineLayoutBase
 
-    PipelineLayoutBase::PipelineLayoutBase(DeviceBase* device,
+    PipelineLayoutBase::PipelineLayoutBase(DeviceBase*,
```

Comment:
nit: Can we have the mDevice / GetDevice combo in PipelineLayoutBase too ?

Reply:
I was thinking the same thing actually, I'll do that.